.travis.yml

```
@@ -49,6 +49,8 @@ before_install:
         pip uninstall -yq numpy
         pip install -i "https://pypi.anaconda.org/scipy-wheels-nightly/simple" --pre "numpy<1.20.0.dev0+20200811"
         pip install -f "https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com" scipy pandas scikit-learn matplotlib h5py Pillow
+        pip install -U git+https://github.com/larsoner/pyvista@mem
+        pip install -U git+https://github.com/larsoner/pyvistaqt@mem
```

Don't make it get the entire `git` history / use git, just use GitHub's URL to fetch an archive
```suggestion
        pip install  https://github.com/larsoner/pyvista/zipball/mem
        pip install https://github.com/larsoner/pyvistaqt/zipball/mem
```